src/components/ErrorMessage/ErrorMessage.test.tsx

```diff
@@ -0,0 +1,45 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import { ErrorMessage } from './ErrorMessage';
+
+describe('ErrorMessage', () => {
+  it('should display text passed to children property as a list', () => {
+    render(
+      <ErrorMessage>
+        <ol>
+          <li>This is an error message</li>
+        </ol>
+      </ErrorMessage>,
+    );
+    expect(screen.getByRole('alertdialog')).toHaveTextContent(
+      'This is an error message',
+    );
+  });
+
+  it('should display text passed with message property', () => {
+    render(<ErrorMessage message='This is an error message' />);
+    expect(screen.getByRole('alertdialog')).toHaveTextContent(
+      'This is an error message',
+    );
+  });
+
+  it('should display text passed message property', () => {
+    render(<ErrorMessage message='This is an error message' />);
+    expect(screen.getByText('This is an error message')).toBeInTheDocument();
+  });
```

Comment:
This test is probably redundant - the above test checks the same thing, just with a better selector since it also checks the role🙂

```suggestion
```